src/OrchardCore.Modules/OrchardCore.AdminTrees/Indexes/AdminTreeIndex.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using OrchardCore.AdminTrees.Models;
+using YesSql.Indexes;
+
+namespace OrchardCore.AdminTrees.Indexes
+{
+    public class AdminTreeIndex : MapIndex
```

Comment:
I don't think you need an index at all. You only have a single document.

Reply:
Done. I removed the index and the migration file too.